models/public/ssd300/ssd300.md

```diff
@@ -23,12 +23,12 @@ See [here](https://github.com/weiliu89/caffe/tree/ssd).
 
 ## Accuracy
 
+The accuracy results were obtained on test data from VOC2012 dataset.
```

Comment:
```suggestion
The accuracy results were obtained on validation data from VOC2012 dataset.
```
test part of dataset is avaliable only on the server

Reply:
Updated